src/tests/routes/test_events.py

```diff
@@ -522,7 +522,7 @@ async def test_fetch_unacknowledged_events(test_app_asyncio, init_test_db, acces
             event_ids = [alert["event_id"] for alert in ALERT_TABLE if alert["device_id"] in devices_group_id]
             events_group_id = [event["id"] for event in EVENT_TABLE if event["id"] in event_ids]
 
-        assert response.json() == [
+        assert response.json()[0] == [
```

Comment:
I don't get how that test can pass? Should this change be reverted?

Reply:
this is because I am now sending List(Events) + List(URL) in a tuple but I cannot predict the  URL so I am just testing the Events parts